chapter_1/ex1.rb

```diff
@@ -0,0 +1,8 @@
+puts "Hello World"
+puts "Hello Again"
+puts "I like typing this"
+puts "this is fun "
```

Comment:
Please make the rest of this match the program listed in http://learnrubythehardway.org/book/ex1.html.  Capitalization and punctuation is important.

We're drilling this to help build your "Attention to Detail" and "Spotting the differences".